sdk/chaos/arm-chaos/test/chaos_examples.spec.ts

```diff
@@ -23,7 +23,7 @@ const replaceableVariables: Record<string, string> = {
   AZURE_CLIENT_ID: "azure_client_id",
   AZURE_CLIENT_SECRET: "azure_client_secret",
   AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
-  SUBSCRIPTION_ID: "azure_subscription_id"
+  SUBSCRIPTION_ID: "88888888-8888-8888-8888-888888888888"
```

Comment:
chaos test case need a cosmosdb resource dependence. So when I generate cosmosdb with the new apiversion, The apiversions of chaos recording files do not match.
So I rerun chaos test cases